### ISSUE TITLE
TST: skipping a gbq test for now

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#14643]
I am xfailing most of these tests for now till we have the testing sorted: https://github.com/pandas-dev/pandas/commit/d9e75c7e724e5f7449c8c57624ce9395c9ffe11a

**Comment by parthea** [comment]
At first glance, it appears that the test only started failing 3 days ago. The test was written based on the assumption that BigQuery's streaming system caches the table schema for up to 2 minutes , which was mentioned in the following stackoverflow post.
http://stackoverflow.com/questions/25279116/cannot-insert-new-value-to-bigquery-table-after-updating-with-new-column-using-s

This assumption is documented in `gbq.py`.
https://github.com/pandas-dev/pandas/blob/7d40f18718ba80e8b8d706eaed2fc4c098686e71/pandas/io/gbq.py#L569

It appears that this assumption is no longer valid.  Another user mentioned a similar issue in [Google BigQuery Issue 191](https://code.google.com/p/google-bigquery/issues/detail?id=191):
"Our team has noticed that updates are taking over 2 minutes now (more like ~10 minutes) as of the past 12 hours."

`pandas.io.gbq.to_gbq()` will fail if users set `if_exists` to `replace` and the BigQuery system cache is greater than 2 minutes (i.e. change in schema is not reflected in BigQuery)

```
    if_exists : {'fail', 'replace', 'append'}, default 'fail'
        'fail': If table exists, do nothing.
        'replace': If table exists, drop it, recreate it, and insert data.
        'append': If table exists, insert data. Create if does not exist.
```

**Comment by jreback** [comment]
ok we can mark this disabled so it's not run by default (but u can still run locally)
maybe also add s comment about the timeout as you describe above

**Comment by parthea** [comment]
I'd like to keep this issue open because this exact issue is currently open on the BigQuery bug tracking site.
https://code.google.com/p/google-bigquery/issues/detail?id=771

**Comment by jreback** [comment]
closed #15381 

let's migrate this (and others) to pandas-gbq
